src/patternfly/components/Touchspin/touchspin.scss

```diff
@@ -0,0 +1,43 @@
+.pf-c-touchspin {
+  // main
+  --pf-c-touchspin__main--m-active--c-form-control--Color: var(--pf-global--Color--100);
+
+  // units
+  --pf-c-touchspin__main--c-input-group--MarginLeft: var(--pf-global--spacer--sm);
+
+  // control
+  --pf-c-touchspin__control--m-threshold--c-button--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-touchspin__icon--FontSize: var(--pf-global--FontSize--xs);
+
+  // form control
+  --pf-c-touchspin--c-form-control--MaxWidth: #{pf-size-prem(70px)};
+
+  display: inline-flex;
+  align-items: center;
+
+  .pf-c-form-control {
+    display: inline;
+    max-width: var(--pf-c-touchspin--c-form-control--MaxWidth);
+    text-align: right;
+  }
+}
+
+.pf-c-input-group + .pf-c-touchspin__units,
+.pf-c-touchspin__units + .pf-c-input-group {
+  margin-left: var(--pf-c-touchspin__main--c-input-group--MarginLeft);
+}
+
+.pf-c-touchspin__control {
```

Comment:
I don't think you need this block anymore? Also do you think we need the `__control` element? If not, I think we can set the font-size on the button as opposed to the icon.

Reply:
I removed this block. I was thinking about still keeping the icon class + setting the font size there in case we make future changes ?

Reply:
That works. Typically I think we would apply font-size to the button to control the size of the thing in it, but the way you've done it works, too. I agree with keeping the icon element/class.